apps/smart-contracts/core/test/unit/Collateral.test.ts

```diff
@@ -1156,20 +1158,44 @@ describe('=> Collateral', () => {
       const expectedBT = amountToWithdraw.mul(USDC_DENOMINATOR).div(parseEther('1'))
       expect(await baseToken.allowance(collateral.address, withdrawHook.address)).to.be.eq(0)
 
-      await collateral.connect(user1).withdraw(amountToWithdraw)
+      await collateral.connect(user1).withdraw(user1.address, amountToWithdraw)
 
       expect(await baseToken.balanceOf(user1.address)).to.eq(userBTBefore.add(expectedBT))
       expect(await baseToken.balanceOf(collateral.address)).to.eq(contractBTBefore.sub(expectedBT))
       expect(await collateral.balanceOf(user1.address)).to.eq(userCTBefore.sub(amountToWithdraw))
       expect(await baseToken.allowance(collateral.address, withdrawHook.address)).to.be.eq(0)
     })
 
+    it('withdraws collateral to recipient if recipient != withdrawer', async () => {
```

Comment:
base token is withdrawn, not collateral
```suggestion
    it('withdraws to recipient if recipient != withdrawer', async () => {
```
and could you find the corresponding 'recipient = withdrawer' case and rename it to this format?